cloudformation/template.yaml

```diff
@@ -451,7 +451,10 @@ Resources:
   ApiGatewayMarketplaceMeteringRole:
     Type: AWS::IAM::Role
     Properties:
-      RoleName: ApiGatewayMarketplaceMeteringRole
+      RoleName: !Join
+        - ''
+        - - !Ref AWS::StackName
```

Comment:
wasn't this [removed](https://github.com/aws-samples/aws-api-gateway-developer-portal/pull/76/commits/1345d5bdaf9873886e204681ae5f902d5e9420e7)?

Reply:
Awkwardly enough, the param StackName was removed, but the macro "AWS::StackName" always resolves to the name of the stack. The problem was that the StackName param wasn't guaranteed to be the stack name...